BaseTools/Source/Python/build/build.py

```diff
@@ -257,19 +257,24 @@ def LaunchCommand(Command, WorkingDir,ModuleAuto = None):
     if Proc.stdout:
         StdOutThread.join()
 
-    # check the return code of the program
-    if Proc.returncode != 0:
-        if not isinstance(Command, type("")):
-            Command = " ".join(Command)
-        # print out the Response file and its content when make failure
-        RespFile = os.path.join(WorkingDir, 'OUTPUT', 'respfilelist.txt')
-        if os.path.isfile(RespFile):
-            f = open(RespFile)
-            RespContent = f.read()
-            f.close()
-            EdkLogger.info(RespContent)
+    RespFile = os.path.join(WorkingDir, 'OUTPUT', 'respfilelist.txt')
+    if os.path.isfile(RespFile):
+        f = open(RespFile)
+        RespContent = f.read().splitlines()
+        f.close()
+
+        EdkLogger.info("Built with Respfile ... %s", WorkingDir)
+
+        for i in range(0, len(RespContent), 2):

```

Comment:
This is new logic without a comment.  What is its purpose?

Reply:
Hi mike, the PR description explains this new logic, do you want me to add it as a comment in the code itself? I'm more than happy to do what is needed :)

Reply:
The purpose of the added loop is not explained in the PR summary.  Please add comment to code and the PR summary